dicthash/test/test_dicthash.py

```diff
@@ -266,3 +267,19 @@ def test_lists_array_tuples_are_equal(self):
 
         self.assertEqual(hash0, hash1)
         self.assertEqual(hash1, hash2)
+
+    def test_store_and_rehash_h5py(self):
+
+        d0 = {
+            'a': 'asd',
+            'b': 0.12,
+            'c': [3, 4, 5],
+            'd': np.array([[3, 4, 5], [3, 4, 5]]),
+            'e': True
+        }
+        hash0 = dicthash.generate_hash_from_dict(d0)
+        h5.add_to_h5('store_and_rehash_h5py.h5', {'d0': d0}, 'w')
+        d0 = h5.load_h5('store_and_rehash_h5py.h5', 'd0')
```

Comment:
to be more explicit: the loaded dictionary should be called `d1`. (also in the line below)